extensions/commands/art/cmd_promote_graph.py

```diff
@@ -0,0 +1,110 @@
+import json
+import os.path
+
+from conan.api.conan_api import ConanAPI
+from conan.api.output import ConanOutput
+from conan.cli.command import conan_command
+from conans.model.recipe_ref import RecipeReference
+from conans.model.package_ref import PkgReference
+from conan.api.model import MultiPackagesList
+from conan.errors import ConanException
+
+from utils import api_request, assert_server_or_url_user_password
+from cmd_server import get_url_user_password
+
+
+def _get_export_path_from_rrev(rrev):
```

Comment:
We might want to do some ArtifactoryConanAPI common to all artifactory commands, these paths seems very reusable (not now, future)